config/nrfconnect/app/flashing.cmake

```diff
@@ -40,8 +41,8 @@ add_custom_command(OUTPUT "${FLASHBUNDLE_FLASHER_PLATFORM}"
     VERBATIM)
 
 add_custom_command(OUTPUT "${FLASHBUNDLE_FIRMWARE}"
-    DEPENDS zephyr/${KERNEL_HEX_NAME}
-    COMMAND ${CMAKE_COMMAND} ARGS -E copy "zephyr/${KERNEL_HEX_NAME}" "${FLASHBUNDLE_FIRMWARE}"
+    DEPENDS zephyr/${HEX_NAME}
+    COMMAND ${CMAKE_COMMAND} ARGS -E copy "zephyr/${HEX_NAME}" "${FLASHBUNDLE_FIRMWARE}"
```

Comment:
It can be either `merged.hex` or `zephyr.elf` (aka `${KERNEL_HEX_NAME}`) depending on whether the bootloader is being built or not. Maybe the command could try those two files? Sth like `copy "zephyr/merged.hex" "${FLASHBUNDLE_FIRMWARE}" || copy "zephyr/${KERNEL_HEX_NAME}" "${FLASHBUNDLE_FIRMWARE}"`?

Reply:
Couldn't get copy || copy to work, maybe there is some cmake magic I'm missing. Instead used a script to do the copy if it exists.